### ISSUE TITLE
Create `convert` script for toggling https/git@

### ISSUE THREAD

**Issue by ClintLiddick** [personalrobotics/pr-rosinstalls#1]

**Comment by psigen** [comment]
What are the reasons not to use just one or the other?

**Comment by mkoval** [comment]
I agree with @psigen. I suggest using `https` because it is the only option for unauthenticated access.

The only problem I've had with `https` in the past is the annoyance of having to constantly re-enter my username and password. However, this is trivially fixed by enabling Git credential caching; i.e.:
```shell
git config --global credential.helper cache
```

**Comment by ClintLiddick** [comment]
We're using `https` and only committing that, but I plan to use this on my own pc for development, and I want ssh access, hence the convert script (will just be a sed command most likely).

**Comment by aaronjoh** [comment]
I gather from this discussion that the readme notes on "use the included converter script" are aspirational instructions? Or does someone have this script ready to go? If we do include such a script I request that it not be called "convert" because that command is taken. I generally prefer ssh but the credential cache is ok. If we don't include this script, then we need to update the readme.
